recipes/libglvnd/meta.yaml

```diff
@@ -0,0 +1,348 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: libglvnd-split
+  version: {{ version }}
+
+source:
+  url: https://gitlab.freedesktop.org/glvnd/libglvnd/-/archive/v{{ version }}/libglvnd-v{{ version }}.tar.bz2
+  sha256: d0e1925a3c9aee0143b8c181ac31d5637c8faa081759c277b8e16c7075612c11
+  patches:
+    - libglvnd_look_for_egl_icd_in_system_location.patch
+
+build:
+  skip: true  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - meson
+    - pkg-config
+    - ninja
+  host:
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-glproto
+
+outputs:
+  - name: libglvnd
+    files:
+      - "lib/libGLdispatch.so.*"
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+    test:
+      commands:
+        # Shared libraries (change in any SOVER means we need to react on run-export)
+        - test -f $PREFIX/lib/libGLdispatch.so.0
+
+  - name: libglvnd-core-devel
+    files:
+      - lib/pkgconfig/libglvnd.pc
+      - include/glvnd
+    test:
+      commands:
+        # The libglvnd-core-devel package is a bootstrap trick
```

Comment:
I don't understand why you'd want something like this. Can you explain more?

Reply:
This is a gold standard for all of RHEL/Fedora, Debian, Ubuntu, etc...

I copied that description from Fedora 40's package description.

This is for the future when a Conda-native HW-accelerated Mesa needs to be built together with libglvnd bindings.
CC @hmaarrfk 

The reason this must be done is that Mesa must not link with the libglvnd libraries (because of internal conflicts) and thus should be separate, but it needs the three headers to build its own `libEGL_mesa.so.0` vendor-specific GLVND library. The resulting decision in RHEL/Fedora, Debian, Ubuntu, etc. is this.

So while we are at it, I'm thinking about the future.

https://packages.fedoraproject.org/pkgs/libglvnd/libglvnd-core-devel/
https://packages.debian.org/sid/libglvnd-core-dev
https://packages.ubuntu.com/noble/libglvnd-core-dev

Reply:
I think we don't need it because of `ignore_run_exports` functionality in conda.

Reply:
Sure thing. @isuruf 

Merged libglvnd-core-devel and libglvnd-devel into libglvnd-devel.

Reply:
@isuruf But does `ignore_run_exports` allow using only the headers and NOT linking to `libGLdispatch.so.0` of libglvnd when building Mesa during conda-build?

Reply:
> But does ignore_run_exports allow using only the headers and NOT linking to libGLdispatch.so.0 of libglvnd when building Mesa during conda-build when Mesa will link to it otherwise?

That's an issue with Mesa. We can fix that in Mesa's build system.

Reply:
OK. I have nothing more to comment on.